packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1141,11 +1155,28 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
     return child;
   }
 
+  /// Whether this route should ignore pointers when it is the current route and
+  /// transitions are in progress.
+  ///
+  /// Pointers always are ignored when [isCurrent] is false (e.g., when a route
+  /// has a new route pushed on top of it, or during a route's exit transition
+  /// after popping). Override this value to also ignore pointers on pages during
+  /// transitions where this route is the current route (e.g., after the route
+  /// above this route pops, or during this route's entrance transition).
+  ///
+  /// This value should generally be true for Cupertino routes.
```

Comment:
nit: use breadcrumb
```
See also:
 ...
```

Reply:
Done; updated it to include a few different breadcrumbs to routes, although they're all on one bullet. Couldn't quite tell from looking at the style guide if that's not preferred